src/models-spec.js

```diff
@@ -56,6 +71,45 @@ describe('models', () => {
           expect(`${jsonData.type.toLowerCase()}.json`).toEqual(file.toLowerCase());
         });
 
+        it('should check sameAs on fields actually exist when they are properties from schema.org', () => {
+          for (const field in jsonData.fields) {
+            if (
+              Object.prototype.hasOwnProperty.call(jsonData.fields, field)
+                && typeof jsonData.fields[field].sameAs === 'string'
+                && jsonData.fields[field].sameAs.match(/^https:\/\/schema.org/)
+            ) {
+              const propertyId = jsonData.fields[field].sameAs.replace(/^https/, 'http');
```

Comment:
Are you still replacing https with http as the changes have not landed into the models yet?

Reply:
Depends which models you mean. The OA model specification (those JSON files in this repo) will remain using https for schema ids but the schema.org JSON-LD still uses http (but this is where it may in the future become https but no idea what schema.org's release timeline is like).

Reply:
Ah - makes sense, LGTM!